lib/mix/test/mix/dep_test.exs

```diff
@@ -488,6 +488,10 @@ defmodule Mix.DepTest do
     Mix.RemoteConverger.register(nil)
   end
 
+  defp sorted_keys(map) do
+    map |> Map.keys() |> Enum.sort()
+  end
```

Comment:
Side note: I'm starting to wonder if a `Map.sorted_keys/1` function would be nice.
It might just be a shorthand for `map |> Map.keys() |> :lists.sort()` first, but:
- more compact
- might be optimized in the future if OTP exposes something more efficient

Reply:
I am waiting on such for now to see if we will have use cases beyond tests.